docs/reference/javascript-api-for-office-error-codes.md

```diff
@@ -76,6 +76,7 @@ The following table lists the error codes, names, and messages displayed, and th
 |8021|Out of range|The column index value is out of the allowed range. Use a positive value (0 or higher) that's less than the number of columns.|The column index is more than the biggest column index of the table or less than 0.|
 |8022|Out of range|The value is out of the allowed range.|Some of the values in the format are out of the supported ranges.|
 |9016|Permission denied|Permission denied|Access is denied.|
+|9020|Generic Response Error|An internal error has occurred.|Refers to an internal error condition.|
```

Comment:
That's a good callout, thanks. It looks like 9020 may be the generic error returned in Outlook on the web, and the 5001 error is returned in Outlook on Windows. I'll update the description for 5001 as well.

Reply:
Upon further investigation, it appears that the 9020 error occurs at the JS layer, so it could occur in other hosts other than Outlook. Thanks for the discussion on this. I've incorporated your suggestion for you to review.